presto-main/src/main/java/com/facebook/presto/connector/system/NodeSystemTable.java

```diff
@@ -44,7 +50,7 @@
             .column("http_uri", VARCHAR)
             .column("node_version", VARCHAR)
             .column("coordinator", BOOLEAN)
-            .column("active", BOOLEAN)
+            .column("state", VARCHAR)
```

Comment:
@dain thoughts on changing this? Should we make this backwards compatible and keep the old field?